### DISCUSSION TITLE
SSL Cloudflare

### DISCUSSION THREAD

**Discussion by fsan2val** [owncast/owncast#3724] [Q&A] [answered]
Hello everyone.

I installed Owncast on a VPS on a subdomain created in cloudflare and activated the Proxy.

My question, is it necessary to make another setting on the server for the SSL certificate?

Because I have a problem, with this setting I cannot transmit from OBS studio, I get that it cannot connect to the server, but if I deactivate the cloudflare proxy, OBS works but my pages are left without a certificate

Someone who can give me light

Thank you

**Comment by gabek** [comment]
I can't speak about your Cloudflare configuration, or what's going on there. But your Owncast server's RTMP service will have to be available to you, so you can connect to it from your broadcasting software. If Cloudflare only supports proxying web traffic, then RTMP isn't web, so you might have to find another way to gain access to your Owncast server if it's hidden behind Cloudflare.

**Reply by fsan2val** [reply]
Thank you very much for your valuable response. In fact, I will no longer use the clouflare proxy, but I am left without SSL, what method do you recommend to install it that is as simple as possible? thank you

**Comment by aaronpk** [comment]
CloudFlare will handle your https traffic, so if you map your domain `example.com` to CloudFlare, which points to your VPS, your viewers can visit `example.com` which will be served by CloudFlare. When you want to stream to the VPS, you need to use an alternative hostname that actually points to your VPS. This hostname would be only for the video stream ingestion, for example `server.example.com`. You can configure the DNS record in CloudFlare from `server.example.com` to your VPS's IP address. Don't give this out publicly, but use that for the RTMP address you're pushing video to.

**Reply by fsan2val** [reply]
Thank you very much also for your time. I tell you that I have created a subdomain, for example. live.example.com on cloudflare but remains without an SSL certificate. My question is how do I put SSL certification in owncast? some simple and easy recommendation to update the certificate thank you

**Comment by fsan2val** [comment, marked as answer]
I was able to solve the problem, I had to restart the server and it works fine for now.

Thanks for everything
